### PR TITLE
Multi-scale testing

### PR DESCRIPTION
**Pull request by fallingdust**
Implemented bbox augmentations(h-flip, multi-scale, multi-scale h-flip) in test time, just like Detectron.

There's a little hack in the data loading process, if you have better solution, please tell me.

### CODE REVIEW

**Comment by zimenglan-sysu-512** [comment]
hi @fallingdust 
can u give some results of augmentations (flip, scales flip+scales)?

**Comment by fallingdust** [comment]
hi @zimenglan-sysu-512 ,
I evaluated on **coco_2014_minival** with pretrained model downloaded from _https://download.pytorch.org/models/maskrcnn/e2e_faster_rcnn_R_101_FPN_1x.pth_, here is the bbox mAP:

Settings|mAP
--------------|--------------
w/o BBOX_AUG|39.1
w/ H_FLIP: True|39.7
w/ SCALES: (400, 600, 1000, 1200) MAX_SIZE: 2000|40.8
w/ H_FLIP: True SCALES: (400, 600, 1000, 1200) MAX_SIZE: 2000 SCALE_H_FLIP: True| 41.1

File: maskrcnn_benchmark/engine/bbox_aug.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import torch
+import torchvision.transforms as TT
+
+from maskrcnn_benchmark.config import cfg
```
Comment (fmassa) [inline maskrcnn_benchmark/engine/bbox_aug.py]: using the config globally in the library is something I've tried to avoid, but let's merge this as is and maybe modify this later on

**Comment by shikunyu8** [comment]
Since you disabled the bbox filtering after box head, you will have 1000 boxes fed into the mask head. How could you fit 1000 masks into GPU memory?

**Comment by shikunyu8** [comment]
I guess you test on faster rcnn, but the config file is mask rcnn.

**Comment by youngwanLEE** [comment]
@fallingdust @shikunyu8 @fmassa 
This TTA is only for box detection?  Is it available for mask rcnn?

**Comment by shikunyu8** [comment]
Hi, @youngwanLEE 
Yes. It's not available for mask rcnn.
